src/Dev/TestSession.php

```diff
@@ -202,44 +201,55 @@ public function sendRequest($method, $url, $data, $headers = null, $session = nu
      * Wrong: array('mycheckboxvalues' => array(1 => 'one', 2 => 'two'))
      * Right: array('mycheckboxvalues[1]' => 'one', 'mycheckboxvalues[2]' => 'two')
      *
-     * @see http://www.simpletest.org/en/form_testing_documentation.html
-     *
-     * @param string $formID HTML 'id' attribute of a form (loaded through a previous response)
-     * @param string $button HTML 'name' attribute of the button (NOT the 'id' attribute)
-     * @param array $data Map of GET/POST data.
-     * @return HTTPResponse
+     * $formID HTML 'id' attribute of a form (loaded through a previous response)
+     * $button HTML 'name' attribute of the button (NOT the 'id' attribute)
+     * $data Map of GET/POST data.
+     * $withSecurityToken Submit with the form's security token if there is one.
```

Comment:
Still need `@param` for IDE to pick these up correctly.

Reply:
Sorry, I should have been clearer about this. The type is also required in the phpdoc.
Here's how VSCode currently interprets this method signature and phpdoc:
![VScode's interpretation of submitForm()](https://user-images.githubusercontent.com/36352093/217095695-2faa5f23-ff3b-4ac5-a12f-1be27b56df31.png)

You can see that it thinks `@param $formID HTML` means there's a `HTML` parameter of type `$formID` as well as a `$formID` parameter of type `string`

The phpdoc for that should start `@param string $formID` and then the description. This unfortunately means there's a double up of the type information in the phpdoc as well as in the method signature itself, but that's necessary if we want to include a description of the parameter in the phpdoc.

Reply:
DONE